index.yaml

```diff
@@ -609,6 +651,20 @@ entries:
     - assets/longhorn/longhorn-1.0.200.tgz
     version: 1.0.200
   longhorn-crd:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: longhorn-system
+      catalog.cattle.io/release-name: longhorn-crd
+    apiVersion: v1
+    created: "2021-09-02T23:33:08.754460728+08:00"
+    description: Installs the CRDs for longhorn.
+    digest: 0da94608691f075178a186a75b2b12b7fe4208ea32fc44dcce7c101fc1fd01d8
+    name: longhorn-crd
+    type: application
+    urls:
+    - assets/longhorn/longhorn-crd-101.0.0+up1.2.0.tgz
+    version: 101.0.0+up1.2.0
```

Comment:
needs to be regenerated after version is changed

Reply:
As I mentioned in https://github.com/rancher/charts/pull/1428#issuecomment-912224971, we should use `100.1.0` instead.